### PR TITLE
Update group-analytics.mdx

### PR DESCRIPTION
**Pull request by kurbycchua**
Clarified that 1M group profile limit is per group key and provided example of why this limit exists. Also per @mavlee NOT to go into details why we limit group profiles to 1M (ie  - when analysing against user level data e.g. break down my user funnel by company group key) - Note to support and CSAs that we would need to explain this to customers if they ask

### CODE REVIEW

File: pages/docs/data-structure/advanced/group-analytics.mdx [hunk 1]
```diff
@@ -8,7 +8,7 @@ Historically, Mixpanel grouped events by a single identifier called the distinct
 
 Group Analytics is available as an add-on package to customers on [Growth](https://mixpanel.com/pricing/) and [Enterprise plans](https://mixpanel.com/pricing/).
 
-Customers on the Growth plan can buy this add-on [online](https://mixpanel.com/pricing/), while customers on the Enterprise plans can reach out to their Customer Success Manager or the [Mixpanel Sales Team](https://mixpanel.com/contact-us/sales/). Online plans allow up to 6 group keys. Enterprise customers can choose between 3 or 6 group keys. Each group can have a total 1M profiles.
+Customers on the Growth plan can buy this add-on [online](https://mixpanel.com/pricing/), while customers on the Enterprise plans can reach out to their Customer Success Manager or the [Mixpanel Sales Team](https://mixpanel.com/contact-us/sales/). Online plans allow up to 6 group keys. Enterprise customers can choose between 3 or 6 group keys. Each group key can have a total of 1M group profiles (see [Limits and Caveats](https://docs.mixpanel.com/docs/data-structure/advanced/group-analytics#limits-and-caveats)).
```
Comment (mavlee) [inline pages/docs/data-structure/advanced/group-analytics.mdx]: relative link

File: pages/docs/data-structure/advanced/group-analytics.mdx [hunk 2]
```diff
@@ -153,7 +153,7 @@ Because a user can be part of multiple groups within a group key, set the value
 ## FAQ
 
 ### Limits and Caveats
-The maximum number of group profiles that may be tracked is 1M profiles. When exceeding this limit, queries that use group properties may fail due to the size of the group table.
+The maximum number of group profiles that may be tracked is 1M profiles per group key. When exceeding this limit, queries that use group profile properties against user level analysis (eg: break down my user funnel by company group key) may fail due to the size of the group table.
```
Comment (mavlee) [inline pages/docs/data-structure/advanced/group-analytics.mdx]: honestly i think we put too much detail here

we should just list our limit and put some justification

File: pages/docs/data-structure/advanced/group-analytics.mdx (hunk 1 above)
Comment (kurbycchua) [reply pages/docs/data-structure/advanced/group-analytics.mdx]: good catch! updated.

File: pages/docs/data-structure/advanced/group-analytics.mdx (hunk 2 above)
Comment (kurbycchua) [reply pages/docs/data-structure/advanced/group-analytics.mdx]: simplified the section

File: pages/docs/data-structure/advanced/group-analytics.mdx [hunk 3]
```diff
@@ -153,9 +153,11 @@ Because a user can be part of multiple groups within a group key, set the value
 ## FAQ
 
 ### Limits and Caveats
-The maximum number of group profiles that may be tracked is 1M profiles. When exceeding this limit, queries that use group properties may fail due to the size of the group table.
 
-No single group identifier (e.g. company_id), can have more than 1M events/day. When a particular group identifier has more events than this limit, query performance will suffer and sometimes fail.
+Below set limits ensure reports using group analytics perform correctly, especially when they are analyzed against user level data (e.g. break down my user funnel by company group key):
+
+- 1M group profiles per group key (e.g. companny_id)
```
Comment (mavlee) [inline pages/docs/data-structure/advanced/group-analytics.mdx]: typo

File: pages/docs/data-structure/advanced/group-analytics.mdx [hunk 4]
```diff
@@ -153,9 +153,11 @@ Because a user can be part of multiple groups within a group key, set the value
 ## FAQ
 
 ### Limits and Caveats
-The maximum number of group profiles that may be tracked is 1M profiles. When exceeding this limit, queries that use group properties may fail due to the size of the group table.
 
-No single group identifier (e.g. company_id), can have more than 1M events/day. When a particular group identifier has more events than this limit, query performance will suffer and sometimes fail.
+Below set limits ensure reports using group analytics perform correctly, especially when they are analyzed against user level data (e.g. break down my user funnel by company group key):
```
Comment (mavlee) [inline pages/docs/data-structure/advanced/group-analytics.mdx]: The limits below ensure reports using group analytics perform correctly.

File: pages/docs/data-structure/advanced/group-analytics.mdx (hunk 3 above)
Comment (kurbycchua) [reply pages/docs/data-structure/advanced/group-analytics.mdx]: good catch.. corrected

File: pages/docs/data-structure/advanced/group-analytics.mdx (hunk 4 above)
Comment (kurbycchua) [reply pages/docs/data-structure/advanced/group-analytics.mdx]: Edited to 

The limits below ensure group analytics works correctly in Mixpanel:
- 1M group profiles per group key (e.g. company_id) - when analysing against user level data (e.g. break down my user funnel by company group key)
- 1M events per day per group identifier (e.g. company_id = "Mixpanel") - see [Hot Shard Limits](/docs/tracking-best-practices/hot-shard-limits)

**Review by mavlee — APPROVED** [review]

**Pull request merged**